pipeline/README.MD

```diff
@@ -2,11 +2,36 @@
 
 This directory contains all the necessary code and documentation for the Extract Transform Load (ETL) pipeline used to collect, clean and upload the data collected from LNMH's Plant Monitoring array.
 
+## Requirements
+
+To run this script, you will need the following:
+
+- **Python**: Version 3.6 or higher
+- `pytest`: For running unit tests
+- `pytest-cov`: For measuring test coverage
+- `requests`: For making HTTP requests
+- `pandas`: For data manipulation and analysis
+- `python-dotenv`: For loading environment variables from a `.env` file
+- `pymssql`: For connecting to Microsoft SQL Server
+- `aiohttp`: For asynchronous HTTP requests
+- `certifi`: For secure SSL/TLS connections
+- `asyncio`: For asynchronous programming
+- `pytest-asyncio`: For testing asynchronous code
+
+To install these dependencies, use the following command:
+
+```zsh
+pip3 install -r requirements.txt
+```
+
+
 ## Files Explained
 - `extract.py` - establishes a connection to the Heroku API to extract plant metrics data generated every minute, ensuring seamless data retrieval for further processing.
 - `test_extract.py` - this test file employs patching techniques to mock external dependencies and validate the functionality of `extract.py`, including the correct extraction of plant metrics, while preventing any real-world API calls. 
 - `transform.py` - this file performs data cleaning tasks, such as removing null values, converting columns to appropriate data types, and ensuring numerical consistency by rounding values to predefined precision levels.
-- `test_transform.py` - this test file verifies the main functions in transform.py through unit tests, achieving 82% test coverage with pytest, and ensuring that data transformation is performed accurately without introducing errors.
+- `test_transform.py` - this test file verifies the main functions in `transform.py` through unit tests, achieving 82% test coverage with pytest, and ensuring that data transformation is performed accurately without introducing errors.
+- `load.py` - this file loads takes clean data from transform and loads it into the Microsoft SQL Server hosted on RDS AWS.
+- `test_load.py` - this test file verifies the core functions in `load.py` through unit tests.
```

Comment:
do you think we should order the files, like keep all the test files at the bottom?